core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -1590,24 +1698,36 @@ pub static mp_module_trezorui2: Module = obj_module! {
     ///     """Show address details - QR code, account, path, cosigner xpubs."""
     Qstr::MP_QSTR_show_address_details => obj_fn_kw!(0, new_show_address_details).as_obj(),
 
+    /// def show_spending_details(
+    ///     *,
+    ///     account: str,
+    ///     fee_rate: str | None = None,
+    /// ) -> object:
+    ///     """Show metadata when for outgoing transaction."""
+    Qstr::MP_QSTR_show_spending_details => obj_fn_kw!(0, new_show_spending_details).as_obj(),
+
     /// def confirm_value(
     ///     *,
     ///     title: str,
-    ///     description: str,
     ///     value: str,
+    ///     description: str | None = None,
+    ///     subtitle: str | None = None,
     ///     verb: str | None = None,
+    ///     verb_cancel: str | None = None,
+    ///     info_button: bool = False,
     ///     hold: bool = False,
     /// ) -> object:
     ///     """Confirm value. Merge of confirm_total and confirm_output."""
     Qstr::MP_QSTR_confirm_value => obj_fn_kw!(0, new_confirm_value).as_obj(),
 
-    /// def confirm_joint_total(
+    /// def confirm_total(
     ///     *,
-    ///     spending_amount: str,
-    ///     total_amount: str,
+    ///     title: str,
+    ///     items: List[Tuple[str, str]],
+    ///     info_button: bool = False,
     /// ) -> object:
-    ///     """Confirm total if there are external inputs."""
-    Qstr::MP_QSTR_confirm_joint_total => obj_fn_kw!(0, new_confirm_joint_total).as_obj(),
+    ///     """Transaction summary."""
```

Comment:
I would probably add info that it is always HoldToConfirm, as that is an important property